docs/source/getting_started/running_at_scale.ipynb

```diff
@@ -23,12 +23,7 @@
     "\n",
```

Comment:
<p>In the first image, circle the Register Algorithm button in the Launcher.</p><p> </p><p>".git file." is confusing it's a url that ends in git, it's not a file</p><p> </p><p>"space you expect" - disk space, including all inputs, scratch, and outputs</p><p> </p><p>"URL of the Stack" workspace image environment</p><p> </p><p>file inputs, are automatically loaded to /inputs in your job</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>fixed in update 04f1bd4e0505977f3f787b3d41be18cfe265d627</p>